VocabularyTrainer.xcodeproj/project.pbxproj

```diff
@@ -652,15 +672,15 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 6T68RPD4WZ;
+				DEVELOPMENT_TEAM = 972LY7PTVT;
 				INFOPLIST_FILE = VocabularyTrainer/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.3.0;
-				PRODUCT_BUNDLE_IDENTIFIER = st.mic.VocabularyTrainer;
+				PRODUCT_BUNDLE_IDENTIFIER = st.mic.VocabularyTrainerr;
```

Comment:
oh what happened here? :D

Reply:
oh I see you changed it probably for testing on device, right?

You can keep it like that, I will change it back once preparing the next release.